cmd/oras/internal/display/metadata/text/manifest_index.go

```diff
@@ -24,6 +24,7 @@ import (
 // ManifestIndexCreateHandler handles text metadata output for index create events.
 type ManifestIndexCreateHandler struct {
 	printer *output.Printer
+	desc    ocispec.Descriptor
```

Comment:
nit: I feel that the name `desc` is a bit ambiguous

Reply:
renamed